crt_portal/cts_forms/templates/forms/report_class.html

```diff
@@ -17,21 +14,23 @@
       </div>
     {% elif field.name == 'other_class' %}
       <div id='other-class-option' class="padding-left-4">
-        {% if field.errors %}
-          {% include "forms/error_warning.html" with errors=field.errors %}
-        {% endif %}
         <div class="blue-left-highlight">
           <div class="padding-left-2">
-          <label for="{{ field.id_for_label }}">{{ field.help_text }}</label>
-          {{ field }}
-          <div class="margin-top-1">
-            {% include "forms/word_counter.html" with word_limit=10 %}
-          <div>
+            <label for="{{ field.id_for_label }}">{{ field.help_text }}</label>
+            {{ field }}
+            <div class="margin-top-1">
+              {% include "forms/word_counter.html" with word_limit=10 %}
+            </div>
           </div>
         </div>
       </div>
     {% endif %}
   {% endfor %}
+  {% for field in wizard.form.visible_fields %}
```

Comment:
Because of the way the Protected Class and Other Reason fields work together, the server-side errors need to go below both fields in order to avoid a visually awkward state where the error alert is sandwhiched between checklist and text box.